QUEST.tsv

```diff
@@ -12,16 +12,16 @@ xQUEST_20150317_000011	Please select a class.
 xQUEST_20150317_000012	Bokor Master
 xQUEST_20150317_000013	So, you want to learn more about this stone slate. {nl}If you wish, I will show you the will of its creator.
 xQUEST_20150317_000014	Goddess Laima
-xQUEST_20150317_000015	By the time you receive this, a great deal of time will have passed. {nl}I am the Goddess of Fate and Wisdom - Laima.
-xQUEST_20150317_000016	When I first made this revelation, 600 years ago, I had witnessed much death and grief. {nl}Your seeing of this message likely means that all that was forseen has come to pass.
-xQUEST_20150317_000017	The Demons have started the first of these calamities, and the goddesses have all but vanished. {nl}I knew all of this in advance and spent great effort to interfere with fate, but my power alone was not enough.
-xQUEST_20150317_000018	I have wandered through many far-seeing dreams to find an answer… {nl}And you were at the end of those dreams… a Savior.
-xQUEST_20150317_000019	That being said, I expected the Demons would never rest while I held the power of foresight. {nl}I chose to hide my form, until the time of your appearance.
+xQUEST_20150317_000015	By the time you see this, a great deal of time will have flowed between us. {nl}I am the Goddess of Fate and Wisdom - Laima.
+xQUEST_20150317_000016	When I first made this revelation, 600 years ago, I had witnessed much death and grief. {nl}Your seeing of this message likely means that all that I foresaw has come to pass.
+xQUEST_20150317_000017	The Demons have started the first blights, and the goddesses have vanished. {nl}I knew all of this in advance and spent great effort to interfere with fate, but my power alone was not enough.
```

Comment:
"Passed" is fine imo. "Blights" gives the connotation of decay and ugliness (like Urban Blight), I'd stick with calamity for end-of-the-world types stuff.